models/pull.go

```diff
@@ -636,3 +636,20 @@ func (pr *PullRequest) updateCommitDivergence(e Engine, ahead, behind int) error
 func (pr *PullRequest) IsSameRepo() bool {
 	return pr.BaseRepoID == pr.HeadRepoID
 }
+
+// GetPullRequestByHeadBranch returns all prs by head branch
+// Since there could be multiple prs to the same head branch, this function returns a slice of prs
+func GetPullRequestByHeadBranch(headBranch string, headRepo *Repository) (prs []*PullRequest, err error) {
+	prs = make([]*PullRequest, 0)
+	err = x.
+		Where("head_branch = ? AND head_repo_id = ? AND status = ?", headBranch, headRepo.ID, PullRequestStatusMergeable).
```

Comment:
I would add PullRequestStatusMergeable as option for GetPullRequestByHeadBranch so this func can be reused

Reply:
Done.